css/desktop.css

```diff
@@ -0,0 +1,228 @@
+@media screen and (min-width: 768px) {
+  /* headline section */
+
+  #page-head {
+    background: transparent;
+    justify-content: center;
+  }
+
+  #headeline-bg {
+    display: block;
+    width: 100%;
+    height: 120vh;
+    position: absolute;
+    z-index: -1;
+  }
+
+  #mobile-bg {
+    display: none;
+  }
+
+  #logo {
+    position: absolute;
+    display: none;
+  }
+
+  .fa-bars {
+    position: absolute;
+    display: none;
+  }
+
+  #menu-items {
+    display: flex;
+    padding-right: 40px;
+    list-style-type: none;
+    width: 100%;
+  }
+
+  #menu-items a {
+    margin-left: 2rem;
+    color: #42526e;
+    font-weight: 600;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  #headline {
+    padding-bottom: 1rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 5rem;
+  }
+
+  #introduction {
+    width: 60%;
+  }
+
+  #introduction h1 {
+    font-size: 48px;
+    line-height: 1.1em;
+    font-weight: 800;
+  }
+
+  #first-break {
+    display: none;
+  }
+
+  #text-section p {
+    font-size: 20px;
+    line-height: 1.2em;
+    padding-bottom: 2rem;
+  }
+
+  #text {
+    text-align: center;
+  }
+
+  #rule {
+    display: none;
+  }
+
+  #break {
+    display: none;
+  }
+
+  /* projects section */
+
+  #projects {
+    margin-top: 13rem;
+  }
+
+  #projects-container {
+    display: flex;
+    justify-content: space-around;
+    width: 81%;
+    flex-wrap: wrap;
+    margin: 0 auto;
+  }
+
+  .project-file {
```

Comment:
Please kindly fix the spacing between your cards in the projects-container and lets stick to the figma design crefully inspect elements. 

Your design:
![Screen Shot 2](https://user-images.githubusercontent.com/79506105/165896460-8e432f58-d35e-4287-a87c-a943280ef887.png)


Figma design:
![Screen Shot1](https://user-images.githubusercontent.com/79506105/165896474-f9434534-3429-4aad-bb59-308322e8972c.png)



carefully follow design guidelines.

Reply:
And kindly use the correct font colors